dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -251,7 +251,7 @@ function shareProject() {
   });
 }
 
-function remixProject() {
+function remixLegacyShare() {
```

Comment:
Renamed to `remixLevelSourceShare` which matches the name of the controller that handles `/c/` routes.